news/2022-08-01-osumonthly-1.md

```diff
@@ -0,0 +1,211 @@
+---
+layout: post
+title: osu!monthly #1
+date: 2022-07-28
+author: Soetch, Blushing, wowcake, 0x84f
+---
+
+This is the new community's news post : osu!monthly (a new one, again)!
+
+![](/wiki/shared/news/2022-08-01-osumonthly-1/banner-july.png)
+
+## Word of the Redaction : 
+It's been years since **osu!weekly's** release has been paused, 6 years since the community didn't have a regular post, or almost not. Today, we're coming in with **__osu!monthly__**, a bigger format. 
```

Comment:
```suggestion

It's been years since **osu!weekly's** release has been paused, 6 years since the community didn't have a regular post. Today, we're coming in with **__osu!monthly__**, a bigger format. 
```

Reply:
Yep, correcting it!